modules/local/selfcomp_mergeblocks.nf

```diff
@@ -0,0 +1,32 @@
+process SELFCOMP_MERGEBLOCKS {
```

Comment:
There's already a bedtools/merge module at https://github.com/nf-core/modules/blob/master/modules/nf-core/bedtools/merge/main.nf
Could you use it instead of making your own ?
It would only need `-c 4,7,8 -o collapse,min,max -delim "|" -d 100000` to be defined in `conf/modules.config`

Reply:
installed bedtools merge and changed the workflow. Thanks Matthieu.